conf.xml.tmpl

```diff
@@ -171,10 +171,13 @@
 		-->
 		<trigger class="org.exist.security.GnuCryptoJceProviderStartupTrigger"/>
 
-		<!--
-		    Trigger for registering eXists XML:DB URL handler with Java
-		-->
-		<trigger class="org.exist.protocolhandler.URLStreamHandlerStartupTrigger"/>
+                <!--
+                    Trigger for registering eXists XML:DB URL handler with Java
```

Comment:
please could you document the difference, why one should choose for one to the other? or explain me that i'll write it for you, but I have no clue yet.....  and the dangers...